test/teiserver/protocols/spring/spring_auth_test.exs

```diff
@@ -478,9 +492,6 @@ CLIENTS test_room #{user.name}\n"
     wreply = _recv_raw(watcher)
     assert wreply == "REMOVEUSER #{user.name}\n"
 
-    # Give it a chance to perform some of the actions
-    :timer.sleep(250)
```

Comment:
Isn't that somewhat still time dependent? I would rather use a polling function to assert `Client.get_client_by_id` returns `nil`